web/packages/teleport/src/JoinTokens/JoinTokens.tsx

```diff
@@ -392,8 +392,8 @@ function TokenDelete({
             {` ${token.safeName}`}
           </Text>
           . This will not remove any resources that used this token to join the
-          cluster. This will remove the ability for any new resources to join
-          with this token and any non-renewable resource from renewing.
+          cluster. This will remove the ability for any new resources or
+          non-renewable resources from joining with this token.
```

Comment:
What is a _non-renewable resource_????

Reply:
The docs explain it more as non-renewable certificates. That is the only place I could find we explain non-renewable but does not use that same phrase. Perhaps we change it to non-renewable certs?

Reply:
@zmb3 please see updated verbiage that refers to resources using non-renewable certs instead of non-renewable resources.